### PR TITLE
added Saudi Arabia bic/swift codes to bank_registry

### PR DESCRIPTION
**Pull request by samizaman**
## Describe your changes
I have added JSON for SWIFT and BIC Codes of Saudi Arabia from https://www.theswiftcodes.com/saudi-arabia/ . I have followed the format mentioned in README.md for the bank registry and ran pre-commit/black on JSON as well.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,9 @@ dist/
 htmlcov/
 .coverage
 test-results.xml
+
+.DS_Store
+
+.idea/
```
Comment (mdomke) [inline .gitignore:18]: Please put system and editor specific ignores in your user's ignore file. E.g.
```
# ~/.gitconfig
[core]
  excludesfile = ~/.gitignore
```
and then 
```
# ~/.gitignore
.DS_Store
.idea
.vscode
```

**Review by mdomke — CHANGES_REQUESTED** [review]
Is there a source from where you generated the contents of the Saudi Arabian registry from, so that we can add it to this project?

**Comment by samizaman** [comment]
> Did you generate the contents of the registry file with a script or collected the information manually? If you have a script can we maybe add it to this project?

I collected the information manually from https://www.theswiftcodes.com/saudi-arabia/

File: .gitignore (hunk 1 above)
Comment (samizaman) [reply .gitignore]: updated it

**Review by mdomke — APPROVED** [review]

**Pull request merged**